src/main/webapp/script.js

```diff
@@ -110,6 +110,719 @@ function createCommentElement(comment) {
+                        },
+                        "url": ":socialhost:/:session_prefix:_/widget/render/ytshare?usegapi\u003d1"
+                    },
+                    "plus": {
+                        "url": ":socialhost:/:session_prefix:_/widget/render/badge?usegapi\u003d1"
+                    },
+                    "family_creation": {
+                        "params": {
+                            "url": ""
+                        },
+                        "url": "https://families.google.com/webcreation?usegapi\u003d1\u0026usegapi\u003d1"
+                    },
+                    "commentcount": {
+                        "url": ":socialhost:/:session_prefix:_/widget/render/commentcount?usegapi\u003d1"
+                    },
+                    "configurator": {
+                        "url": ":socialhost:/:session_prefix:_/plusbuttonconfigurator?usegapi\u003d1"
+                    },
+                    "zoomableimage": {
+                        "url": "https://ssl.gstatic.com/microscope/embed/"
+                    },
+                    "appfinder": {
+                        "url": "https://gsuite.google.com/:session_prefix:marketplace/appfinder?usegapi\u003d1"
+                    },
+                    "savetowallet": {
+                        "url": "https://pay.google.com/gp/v/widget/save"
+                    },
+                    "person": {
+                        "url": ":socialhost:/:session_prefix:_/widget/render/person?usegapi\u003d1"
+                    },
+                    "savetodrive": {
+                        "url": "https://drive.google.com/savetodrivebutton?usegapi\u003d1",
+                        "methods": ["save"]
+                    },
+                    "page": {
+                        "url": ":socialhost:/:session_prefix:_/widget/render/page?usegapi\u003d1"
+                    },
+                    "card": {
+                        "url": ":socialhost:/:session_prefix:_/hovercard/card"
+                    }
+                }
+            },
+            "h": "m;/_/scs/apps-static/_/js/k\u003doz.gapi.en.yyhByYeMTAc.O/am\u003dwQc/d\u003d1/ct\u003dzgms/rs\u003dAGLTcCN9qAMm_5_ztFCxaPySR5cb8QjKkw/m\u003d__features__",
+            "u": "https://apis.google.com/js/api.js",
+            "hee": true,
+            "fp": "50939a1cffc34695362a4bdc910d88f4e983656d",
+            "dpo": false
+        },
+        "fp": "50939a1cffc34695362a4bdc910d88f4e983656d",
+        "annotation": ["interactivepost", "recobar", "signin2", "autocomplete", "profile"],
+        "bimodal": ["signin", "share"]
+    }
+});
+
+function authenticate() {
+  return gapi.auth2.getAuthInstance()
+      .signIn({scope: "https://www.googleapis.com/auth/youtube.readonly"})
+      .then(function() { console.log("Sign-in successful"); },
+            function(err) { console.error("Error signing in", err); });
+}
+
+function loadClient() {
+  gapi.client.setApiKey("AIzaSyCoTpMozat1rLnBqHPzd2GN4e5NE3al5w8");
```

Comment:
Do we need to keep these API keys private?

Reply:
yes :(
fixed in https://github.com/googleinterns/step20-2020/pull/84